helm/sematic/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: sematic
+version: 0.1.0
+kubeVersion: ">=1.21.0-0"
+description: Build pipelines for prototype to production ML in days not weeks
```

Comment:
The one-liner so far was
```
Prototype-to-production ML in days not weeks
```
but I am thinking of changing it to
```
The open-source Continuous Learning Machine
```

but it may not matter in this file.